src/services/disableUserService.js

```diff
@@ -0,0 +1,40 @@
+import Database from './gateway/database'
+import AuthService from './authService'
+
+export default function DisableUserService() {
+  const isUserDisabled = userUid => {
+    const ref = Database('disabledUsers')
+    return ref.child(userUid).once('value').then(user => {
+      return user.exists()
+    })
+  }
+
+  return {
+    isUserDisabled: isUserDisabled,
+
+    blockUser: userUid => {
+      return Database('users').child(userUid).once('value')
+        .then(user => {
+          if (!user.exists()) {
+            return Promise.reject(new Error('That userUid is not disabled'))
```

Comment:
No sería "That userUid does not exist" en este caso?

Reply:
👍